### ISSUE TITLE
WCAG 2.1 Quickref fails 1.3.1 and 4.1.1

### ISSUE THREAD

**Issue by dd8** [w3c/wcag#1143]
If you run https://www.w3.org/WAI/WCAG21/quickref/ through the W3 validator it produces the following errors

Error: Duplicate attribute data-toggle 
- a failure of 4.1.1 normative text (though very unlikely to cause accessibility issues)

Error: The aria-controls attribute must point to an element in the same document.
- likely failure of 1.3.1 since `aria-controls="#sidebar"` is supposed to represent the visual relationship of the show/hide button to the sidebar (it should be `aria-controls="sidebar"` )

**Comment by alastc** [comment]
I'm not sure if @yatil is the person to tag in this, or someone else?

**Comment by yatil** [comment]
This should be duplicated or at least noted in the [Quickref repository](https://github.com/w3c/wai-wcag-quickref/) repository.

I don’t know who is the product owner of that project after I have left WAI in March.
